contracts/builtin/WithAdmin.sol

```diff
@@ -16,16 +14,8 @@ contract WithAdmin {
     }
 
     function changeAdmin(address newAdmin) external onlyAdmin {
-        pendingAdmin = newAdmin;
-
-        emit AdminChanging(newAdmin);
-    }
-
-    function acceptAdmin() external {
-        require(msg.sender == pendingAdmin, "E03");
-
-        emit AdminChanged(admin, pendingAdmin);
-        admin = pendingAdmin;
-        pendingAdmin = address(0);
+        require(newAdmin != address(0),"E09");
+        admin = newAdmin;
+        emit AdminChanged(admin, newAdmin);
```

Comment:
Issue: here the `admin` and `newAdmin` will be the same address when emitting event.